bin/fluentd-windows-cmd

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+require 'win32/event'
+require 'optparse'
+
+$LOAD_PATH.unshift(File.join(__dir__, 'lib'))
+require 'fluent/command/windows_cmd'
+
+Fluent::WindowsCmd.new.call

```

Comment:
Should we raise an error when using non-Windows environment?

Reply:
I renamed the command to fluentdctl and support also UNIX environment, it just only sends a UNIX signal to a specified process though.